package.json

```diff
@@ -44,7 +44,7 @@
     "start:angular": "yarn --cwd test/e2e/angular-app start & wait-on http-get://localhost:4200",
     "pretest:e2e": "yarn install:react && yarn install:angular && node pretest.js",
     "test:e2e": "yarn build:release && yarn start:react && yarn start:angular && grunt test-e2e",
-    "protractor": "webdriver-manager update && protractor target/e2e/conf.js",
+    "protractor": "webdriver-manager update --versions.standalone=3.141.59 --gecko false && protractor target/e2e/conf.js",
```

Comment:
Need to pin the version of selenium standalone due to an existing issue with protractor + selenium 4.x release.